wolfssl/internal.h

```diff
@@ -3530,8 +3564,8 @@ struct WOLFSSL_X509 {
 #endif /* WOLFSSL_QT || OPENSSL_ALL */
     int              notBeforeSz;
     int              notAfterSz;
-    byte             notBefore[MAX_DATE_SZ];
-    byte             notAfter[MAX_DATE_SZ];
+    WOLFSSL_ASN1_TIME notBefore;
```

Comment:
There is a bit of a mess above with notAfterTime, notBeforeTime, notBeforeSz and notAfterSz. In my PR #2462 I did a cleanup of this, since I also needed the notBefore/notAfter stuff. Its well tested with new API cases too. This could also get resolved once my PR is merged.

Reply:
For now would the best action be to wait for PR #2462 to be merged into master, or should I resolve these issues in this branch?

Reply:
I’ll go ahead and accept these changes and resolve the fixups on my branch.